.github/workflows/cherry-pick-v7-to-v6.yml

```diff
@@ -1,17 +1,17 @@
-name: Cherry pick master to v6
+name: Cherry pick v7 to v6
 
 on:
   pull_request_target:
     branches:
-      - master
+      - v7.x
     types: ['closed']
 
 permissions: {}
 
 jobs:
-  cherry_pick_to_v6:
+  cherry_pick_v7_to_v6:
```

Comment:
Not of a big issue I'm afraid this may result in false positives if we don't manually remove `needs cherry-pick` when merging CP's PR (because all the labels are copied to the CP's one), we might want to add another contains:
```tsx
contains(github.event.pull_request.labels.*.name, 'v6.x')
```

Otherwise any PR targetted for master with `needs cherry-pick` will open as `v7.x` target.
That when merged (if `needs cherry-pick` label wasn't manually removed) will be opened in `v6.x` undesirably.

What do you think?

Reply:
Or maybe adding a small custom snippet to remove `needs cherry-pick` label from the CP'd PR would be a solution?

Reply:
That's a great point. 👍 
However, I don't know if this is a big problem. 🤔 
We can close the false-positive PRs if they happen.
Besides, these actions might be gone in a few days. 🙈

Reply:
Cool yeah, no immediate issue. We can get back to this later. 👍

Reply:
> Or maybe adding a small custom snippet to remove needs cherry-pick label from the CP'd PR would be a solution?

That would be cleaner, but, sadly the action does not seem to support it. 🤔 
We could either extend the action or create a new one, which would clean-up a PR created with `cherry-pick` to remove the `needs cherry-pick` label if it would become a problem. 🙈 🤷